data/sql/updates/pending_db_world/rev_1614780210274517793.sql

```diff
@@ -0,0 +1,271 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1614780210274517793');
+
+-- Rewrite positions for minerals for zone: Bloodmyst Isle
+
+-- Before: 3 locations
+-- After: 122 locations
```

Comment:
Please avoid including stuff into the SQL update that either are already on the PR or "can be" placed in the PR.
The other short comments below to let the lector know about the intention of the query is ok.

Reply:
OK, I removed 2 of those lines.